### DISCUSSION TITLE
Font characters displayed as squares

### DISCUSSION THREAD

**Discussion by Joci78** [emqx/MQTTX#1423] [General]
Dear community,

I have a Kubuntu 23.04. Using MQTT explorer, I find that when I want to enter the CERT files, the window does not display the characters well. How can I fix this?

-> □□□□□□ □□ □□□□□ □□□

**Comment by ysfscream** [comment]
Hello @Joci78,

Firstly, I would like to point out that you are currently asking in the MQTTX community, while your question appears to be regarding MQTT Explorer. Although both pieces of software are related to MQTT, they are developed by different teams. To receive more specialized assistance, I would recommend posing your question in the official community or GitHub repository for MQTT Explorer.

However, I will still try my best to assist you with the potential solution.

The issue you’re facing seems to be a character encoding or font issue. On Kubuntu 23.04, you might try updating or changing your system's font settings to fix this problem. Here are some steps you might take:

1. **Update Your System** - Ensure your system and all packages are up-to-date by running the following commands:
   
   ```
   sudo apt update
   sudo apt upgrade
   ```

2. **Check Font Settings** - You might try changing your font settings to see if that resolves the issue. Look for font or display options in your system settings to adjust it.

3. **Reinstall MQTT Explorer** - Sometimes, reinstalling the software can resolve quirky issues. Try uninstalling MQTT Explorer and then reinstalling it to see if that resolves the issue.

4. **Check the Logs** - Try to look at the MQTT Explorer logs to see if there are any errors or warnings that might be of assistance.

If you have tried the above methods and the issue persists, I would recommend reaching out to MQTT Explorer support or community for assistance directly.

Good luck!

**Reply by Joci78** [reply]
Dear ysfscream,

First of all, I would like to thank you for your patience and attitude. I misspelled the name, I really have a problem with MQTTX, although this phenomenon also occurs with MQTT Explorer.

My system is up to date. I also tried reinstalling. Unfortunately, it didn't help. Exactly which font setting can affect the operation? Note: Most places are currently in Noto Sans. I changed everything I could, but that didn't help either. :(

The problem only occurs when browsing the file.

Best regards,
Joci78
